Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+@Library('jenkins-devops@refs/tags/7') _
```

Comment:
Does it work using this syntax or do you have to use the syntax we have in other repositories :thinking: 

`@Library('jenkins-devops@7^{}') _
`

Reply:
The latest, I think the syntax worked for `kafka-sanbox` it wasn't triggered automatically though as far as I can remember though.